### DISCUSSION TITLE
image V3: Conversions tilting (some) images from portrait to landscape

### DISCUSSION THREAD

**Discussion by ITSYV** [spatie/laravel-medialibrary#3546] [Q&A]
Since the update to image v3 we experience problems with conversions.
The original picture is saved correctly in portrait mode, but conversions (sometimes) seem to be tilted to landscape mode.
It happens on some pictures, not all. Not sure if this has something to do with the exif data automatically tilting the original?

We're using the imagick driver

**Comment by freekmurze** [comment]
Please PR a failing test so we can investigate this further.

**Reply by ITSYV** [reply]
Not at my dev pc atm, but I can already give you the exif data of one of the tilting pictures
Portrait pictures without the exif data don't seem to be affected

`
Make | Apple
-- | --
Model | iPhone 11 Pro
Orientation | right-top
XResolution | 72
YResolution | 72
ResolutionUnit | inches
Software | 16.5.1
DateTime | 2023:06:25 08:02:24
HostComputer | iPhone 11 Pro
YCbCrPositioning | centered
Exif IFD Pointer | 236
ExifVersion | 0232
`

**Comment by ITSYV** [comment]
Pr submitted https://github.com/spatie/laravel-medialibrary/pull/3553
